cmd/swapd/main_test.go

```diff
@@ -148,7 +148,8 @@ func Test_expandBootnodes(t *testing.T) {
 
 func TestDaemon_PersistOffers(t *testing.T) {
 	defaultXMRMakerSwapdEndpoint := fmt.Sprintf("http://localhost:%d", defaultXMRMakerRPCPort)
-	startupTimeout := time.Second * 9
+	// TODO: figure out a way to tell if the node is fully started, like startedCh
+	startupTimeout := time.Second * 45
```

Comment:
Maybe it should be a defect (and test failure) if our server fails to start in 20 seconds even on a slow VM? Have we looked into or need a ticket to look into what is causing the startup delays?

Reply:
I updated this test to wait on a `startedCh` which is closed when the daemon starts up, seems to work better now.